recipes/breakpad/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+import os
+import textwrap
+
+required_conan_version = ">=1.33.0"
+
+class BreakpadConan( ConanFile ):
+    name = "breakpad"
+    description = "A set of client and server components which implement a crash-reporting system"
+    topics = ["crash", "report", "breakpad"]
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/breakpad/breakpad/"
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    options = {
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "fPIC": True
+    }
+    _env_build = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
```

Comment:
Does it build on Windows?

I tried to build it with `gyp`, but I couldn't get it working.

Reply:
No idea :)